samples/sensor/bq34110/README.rst

```diff
@@ -0,0 +1,42 @@
+.. _bq34110-sample:
+
+BQ34110 Sensor Sample
+#####################
+
+Overview
+********
+
+This sample application retrieves all the fuel gauge parameters,
+- Voltage, in volts
+- Average current, in amps
+- Gauge temperature
+- State of charge measurement in percentage
+- Full Charge Capacity in mAh
+- Remaining Charge Capacity in mAh
+- Average power in W
+- Time to full in minutes
+- Time to empty in minutes
+
+from BQ34110 sensor and prints this information to the UART console.
+
+Requirements
+************
+
+- frdm_k64f board with BQ34110 sensor `BQ34110 Sensor`_
```

Comment:
That's not true; is it? All you need is a board with this sensor in the devicetree, right?

Reply:
yeah, it should work with any platform featuring I2C peripheral.
README.rst file updated